learning/dashboard/webserver/readme.md

```diff
@@ -0,0 +1,161 @@
+
+You can use more than one domain with your Nginx server thanks to Nginx server blocks. The steps below must be followed in order to set up two domains, domainone.com and domaintwo.com, on the same Nginx server:
+
+Create a directory, one for each domain:
+
+Nginx only creates one server block by default, and it points to the /var/www/html directory. If you simply had one site, this would function flawlessly. All of the website's content is housed in the html subdirectory. For both of your domains, create the html subdirectories.
+
+sudo mkdir -p /var/www/domainone.com/htmlsudo mkdir -p /var/www/domaintwo.com/html
+
+Step 1: Change ownership of the directories:
+Doing this will enable the user to create and edit the contents of these new directories.
+sudo chown -R $user:$user /var/www/domainone.com/htmlsudo chown -R $user:$user /var/www/domaintwo.com/html
+
+Step 2: Modify permissions for both the domain directories:
+This step is important to make sure that you don’t need root privileges to make changes to any of the sites.
+sudo chmod -R 755 /var/www
+
+Step 3: Create sample web pages for each domain/site:
+Create a landing page index.html for both domains. Create a basic HTML snippet for a welcome page for both sites.
+nano /var/www/domainone.com/html/index.html
+Once the editor is open, paste the following into the editor:
+<html> <head> <title>Welcome to ServerMania’s Domain One!</title> </head> <body> <h1>You have reached Nginx via Server Block #1</h1> </body></html>
+Save the index.html file and create a similar one for the second domain. Make sure you change the welcome message.
+
+Step 4: Create Server Blocks:
+As mentioned earlier, by default, Nginx has only one server block. You can find it here — /etc/nginx/sites-available/default. So, we'll need two server blocks because we have two sites to support. The server restricts access to port 80. For our new server blocks, let's replicate the default server block.
+sudo cp /etc/nginx/sites-available/default /etc/nginx/sites-available/domainone.com
+Similarly, copy the default file and create the second server block too.
+
+Step 5: Mark One of the Two Servers as Default:
+You can only have a single default server with the default_server option enabled. Please remove it from one of the server blocks.
+
+Step 6: Enable Server Blocks & Restart Nginx:
+You can enable the server blocks by creating symbolic links from the server block configuration files to the sites-enabled directory.
+sudo ln -s /etc/nginx/sites-available/domainone.com /etc/nginx/sites-enabled/
+
+Nginx will begin handling requests for both domains from the appropriate blocks that are received on port 80 as soon as you create symbolic links for both domains and restart Nginx.
+
+You can try visiting both domains to test the modifications and check to see if the contents of index.html are rendered correctly.
+The installation of Nginx on Ubuntu 20.04 can now begin.
+
+
```

Comment:
Please add a list of references for the content if you have used existing tutorials or blogs.